### ISSUE TITLE
Regarding the settings of multi-scale training and loss function weights

### ISSUE THREAD

**Issue by woshixyr** [LianpoWang/Stereo-DICNet2#19]
Are the weights of the two components in the loss function (0.6 and 0.4) set based on experience or obtained through experimental optimization?
